ext/mbstring/mbstring.c

```diff
@@ -3348,7 +3348,7 @@ try_next_encoding:;
 	}
 
 	for (size_t i = 0; i < length; i++) {
-		array[i].demerits *= array[i].multiplier;
+		array[i].demerits *= (uint64_t) array[i].multiplier;
```

Comment:
Should `array[i].multiplier` be rounded before conversion? @alexdowad, @youkidearitai, thoughts?

Reply:
Well, to make the [test](https://github.com/php/php-src/actions/runs/12212106167/job/34070184398?pr=17076#step:13:136) pass, probably
```suggestion
		array[i].demerits = (uint64_t) (array[i].demerits * array[i].multiplier);
```